src/pytest_flyte/plugin.py

```diff
@@ -8,10 +8,10 @@
 from flytekit.clients import friendly
 from pytest_docker.plugin import DockerComposeExecutor
 
-from jinja2 import Environment, PackageLoader
+from jinja2 import Environment, FileSystemLoader
 
 PROJECT_ROOT = os.path.dirname(__file__)
-TEMPLATE_ENV = Environment(loader=PackageLoader("pytest_flyte"))
+TEMPLATE_ENV = Environment(loader=FileSystemLoader(os.path.join(PROJECT_ROOT, "templates")))
```

Comment:
This should not be necessary, if you have the plugin installed in a venv right?

Reply:
@jeevb are you able to reproduce the error I'm getting? I'm using a venv but will try on a fresh one to make sure.

>  Let's try adding an __init__.py file to templates/?

Tried it and that didn't seem to help :(

Reply:
looks this like issue is related: https://github.com/pytest-dev/pytest/issues/5392